codescan/parser.go

```diff
@@ -1467,17 +1467,34 @@ func (ss *setOpResponses) Parse(lines []string) error {
 }
 
 func parseEnum(val string, s *spec.SimpleSchema) []interface{} {
-	list := strings.Split(val, ",")
-	interfaceSlice := make([]interface{}, len(list))
-	for i, d := range list {
-		v, err := parseValueFromSchema(d, s)
```

Comment:
How about users who created their docstrings for enum manually, not using json?

Reply:
I can change the implementation so that when JSON decoding fails it will fall back to the old version. I also recommend a deprecation warning when this happens. Are there already such warnings so I can display them in a similar manner? Any recommendation for it?

Reply:
It sems, I also forgot some debug output, will remove that too.

Reply:
example of  a warning: https://github.com/go-swagger/go-swagger/blob/f1c27cb213c57d0953a020aa87ab87db3f52ddbc/codescan/schema.go#L228

Reply:
Thanks for the example. The change is done. I also added the old test cases and adjusted the tests.